tests/components/aqara/test_util.py

```diff
@@ -0,0 +1,17 @@
+"""Tests for the Aqara switch device."""
+# from unittest.mock import patch
```

Comment:
Commented out import

Reply:
Unresolved this comment, as there is still stale code here.

Reply:
removed